packages/hardhat-zksync-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@matterlabs/hardhat-zksync-node",
-  "version": "0.0.1-beta.7",
+  "version": "0.0.1",
```

Comment:
Are we sure this version is correct? Let me double check

Reply:
I've manually deleted this change